CHANGELOG.md

```diff
@@ -6,6 +6,40 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.2.0] 2019-10-21
+
+### Added
+
+- Added a new [public plugin interface](pkg/secretless/plugin) for building connector plugins
+- Added a new [public log interface](pkg/secretless/log) for standardizing logging
+- Added reporting on code coverage data to unit test output
+- Added ability to run k8s-demo test on GKE
+
+### Changed
+
+- Refactored existing connectors to use new public connector plugin interface
+- Changed the core proxy and plugin manager to support the new public connector
+  plugin interface
+- Edited website Google Group links to link to Discourse
+- Updated the [example plugin](test/plugin) to implement the new plugin interface
```

Comment:
Wouldn't this be covered by change of the interface itself since this is technically just an integration test of that?

Reply:
Another thing that I think is worth calling attention to - the example plugin. No harm in leaving it, imo.